README.md

```diff
@@ -1 +1,12 @@
-# jv-homework-template
+# Hibernate relations
+
+
+Your task is to implement the following steps:
+- add required hibernate dependencies
+- create `hibernate.cfg.xml` file
+- complete implementation of all classes in the `mate/academy/hibernate/relations/model` package
+- complete implementation of all classes in the `mate/academy/hibernate/relations/dao/impl` package
+- create your custom unchecked DataProcessingException and throw it in the catch block at dao layer.
+- complete implementation of all classes in the `mate/academy/hibernate/relations/service/impl/` package
+- make `mate/academy/hibernate/relations/Main.java` working (means be able to run main method without any errors)
```

Comment:
make `mate/academy/hibernate/relations/Main.java` work (you should be able to run `main()` method without any errors)